cmds/webboot/webboot.go

```diff
@@ -80,8 +80,19 @@ func (d *DownloadOption) exec(uiEvents <-chan ui.Event, network bool) (menu.Entr
 			return nil, err
 		}
 	}
-
-	fpath := filepath.Join("/tmp", filename)
+	// If the cachedir is not find, downloaded the iso to /tmp, else create a Downloaded dir in the cache dir.
+	var fpath string
+	if cacheDir == "" {
+		fpath = filepath.Join("/tmp", filename)
```

Comment:
suggestion: instead of /tmp, use os.TempDir, that way your files won't conflict if you run this twice?